### ISSUE TITLE
IP Units Humidity Indicating Type

### ISSUE THREAD

**Issue by arif-hanif** [NREL/OpenStudio#2176]
There is a issue with Humidity indicating condition at maximum dry bulb remaining in SI units when the units for the UI are set to IP.

**Comment by DavidGoldwasser** [comment]
@kbenne what severity should this be, and do you want it tagged for 1.12.0

**Comment by jmarrec** [comment]
While we're at it, the "Humidity Indicating Type" stays blank, when it's clearly present in my ddy file:

![screen shot 2017-09-28 at 3 36 55 pm](https://user-images.githubusercontent.com/5479063/30969521-f10cd64a-a462-11e7-80ff-561555212bb5.png)

```
 SizingPeriod:DesignDay,
  Mannheim Ann Htg 99.6% Condns DB,     !- Name
          1,      !- Month
         21,      !- Day of Month
  WinterDesignDay,!- Day Type
       -8.6,      !- Maximum Dry-Bulb Temperature {C}
        0.0,      !- Daily Dry-Bulb Temperature Range {C}
 DefaultMultipliers, !- Dry-Bulb Temperature Range Modifier Type
           ,      !- Dry-Bulb Temperature Range Modifier Day Schedule Name
    Wetbulb,      !- Humidity Condition Type
       -8.6,      !- Wetbulb at Maximum Dry-Bulb {C}
           ,      !- Humidity Indicating Day Schedule Name
           ,      !- Humidity Ratio at Maximum Dry-Bulb {kgWater/kgDryAir}
           ,      !- Enthalpy at Maximum Dry-Bulb {J/kg}
           ,      !- Daily Wet-Bulb Temperature Range {deltaC}
    100177.,      !- Barometric Pressure {Pa}
       6.71,!- [N/A] Wind Speed {m/s} traditional 6.71 m/s (15 mph)
          0,   !- [N/A] Wind Direction {Degrees; N=0, S=180}
         No,      !- Rain {Yes/No}
         No,      !- Snow on ground {Yes/No}
         No,      !- Daylight Savings Time Indicator
  ASHRAEClearSky, !- Solar Model Indicator
           ,      !- Beam Solar Day Schedule Name
           ,      !- Diffuse Solar Day Schedule Name
           ,      !- ASHRAE Clear Sky Optical Depth for Beam Irradiance (taub)
           ,      !- ASHRAE Clear Sky Optical Depth for Diffuse Irradiance (taud)
       0.00;      !- Clearness {0.0 to 1.1}
````

**Comment by jmarrec** [comment]
The original issue reported has been fixed in #2815, the "Humidity indicating condition at maximum dry bulb" is now switchable from SI to IP:

SI:

![2176_humidity_si](https://user-images.githubusercontent.com/5479063/31218951-24c3a604-a9bc-11e7-977e-01cfee598a95.png)

IP:

![2176_humidity_ip](https://user-images.githubusercontent.com/5479063/31218961-2a144082-a9bc-11e7-9c9c-1d37fd002dd3.png)


The Humidity Indicating Type is still blank, but that is probably to be addressed in #2814
